_includes/meta.html

```diff
@@ -2,8 +2,47 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 
-<title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
-<meta name="description" content="{{ site.description }}">
+<!-- Twitter Card -->
+{% if page.twitter_card == "large" %}
+<meta name="twitter:card" content="summary_large_image">
```

Comment:
small nit, the other if/else blocks indent the tags inside, can we indent this too?